src/http-client.ts

```diff
@@ -46,6 +67,11 @@ export default class FetchHttpClient implements IHttpClient {
     return await this.rawGet<IBanditParametersResponse>(url);
   }
 
+  async getPrecomputedFlags(): Promise<IPrecomputedFlagsResponse | undefined> {
+    const url = this.apiEndpoints.precomputedFlagsEndpoint();
+    return await this.rawGet<IPrecomputedFlagsResponse>(url);
```

Comment:
🟡 Quick thing I just noticed - the endpoint is a `POST`

Reply:
Yep, I bumped the prerelease version with this change https://github.com/Eppo-exp/js-sdk-common/pull/137 and the requests work now when I run the client in a script. It'll work in the browser after a CORS fix